.travis.yml

```diff
@@ -2,10 +2,10 @@ language: php
 
 matrix:
     include:
-        - php: 7.1
-          env: deps=low
         - php: 7.2
+          env: deps=low
         - php: 7.3
+        - php: 7.4snapshot
```

Comment:
Is `7.4` available yet?

Reply:
I've tried to look two days ago and it wasn't yet.

Reply:
https://travis-ci.community/t/php-7-4-release/6197/4

What would you like to do?

Reply:
I think `php: 7.4` is working now.

Reply:
I’ll try later.

Reply:
`7.4` is ok, everything's good, we can merge it! :tada: